README.md

```diff
@@ -316,6 +316,27 @@ The renderField mixin can be called in your template with the field to render as
 {{/fields}}
 ```
 
+#### conditionally rendering fields
+
+`renderField` supports conditionally omitting fields if `useWhen` is passed in field config.  `useWhen` accepts another field key `String` and checks the value is `true`, or an `Object` with the keys `field` and `value`.  The field to check cannot appear on the same step - consider using the `toggle` property to show/hide a field on the same step.
+
```

Comment:
Maybe need to make a point about that...

Such as:

### Use-case:
"Say you have two or more pages, and field x on step one is true, then field z on step two will not be displayed"

And give the above statement a code example